pkg/util/common_test.go

```diff
@@ -0,0 +1,70 @@
+package util
+
+import (
+	"fmt"
+	"reflect"
+	"sort"
+	"testing"
+)
+
+type sigMultiCluster struct {
+	Name          string
+	Status        string
+	LatestVersion string
+}
+
+func (smc sigMultiCluster) String() string {
+	return fmt.Sprintf("%s/%s/%s", smc.Name, smc.Status, smc.LatestVersion)
+}
+
+var (
+	kubefed = sigMultiCluster{
+		Name:          "kubefed",
+		Status:        "archived",
+		LatestVersion: "v0.9.2",
+	}
+	karmada = sigMultiCluster{
+		Name:          "karmada",
+		Status:        "active",
+		LatestVersion: "v1.6.0",
+	}
+)
+
+func TestDiffKey(t *testing.T) {
```

Comment:
Can this test achieve condition coverage? Consider the boundary conditions.

Reply:
done